src/components/CommentField.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useRef, useState, FormEvent, useEffect } from 'react';
+import { EnrichedActivity } from 'getstream';
+
+import { Avatar } from './Avatar';
+import { Button } from './Button';
+import { Textarea, TextareaProps } from './Textarea';
+import { inputValueFromEvent } from '../utils';
+import { useTranslationContext, FeedManager } from '../Context';
+import { DefaultAT, DefaultUT } from '../Context/StreamApp';
+
+export type CommentFieldProps<UT extends DefaultUT = DefaultUT, AT extends DefaultAT = DefaultAT> = {
+  activity: EnrichedActivity<UT, AT>;
+  onAddReaction: FeedManager<UT, AT>['onAddReaction'];
+  image?: string;
+  kind?: string;
+  onSuccess?: () => void;
+  placeholder?: string;
+  trigger?: TextareaProps['trigger'];
+};
+
+export const CommentField = <UT extends DefaultUT = DefaultUT, AT extends DefaultAT = DefaultAT>({
+  activity,
+  onAddReaction,
+  onSuccess,
+  image,
+  placeholder,
+  trigger,
+}: CommentFieldProps<UT, AT>) => {
+  const { t } = useTranslationContext();
+  const textareaReference = useRef<HTMLTextAreaElement>();
+  const [text, setText] = useState<string>();
+
+  const handleFormSubmit = async (event: FormEvent<HTMLFormElement> | KeyboardEvent) => {
+    event.preventDefault();
+
+    if (!text) return;
+
+    try {
+      await onAddReaction('comment', activity, {
+        text,
+      });
+    } catch (error) {
+      console.error(error);
+    }
+
+    setText('');
+    onSuccess?.();
+  };
+
+  const handleFormSubmitKey = (event: KeyboardEvent) => {
+    const { current: textarea } = textareaReference;
+    if (event.key === 'Enter' && textarea?.nextSibling === null) {
+      handleFormSubmit(event);
+    }
+  };
+
+  useEffect(() => {
+    if (!textareaReference.current) return;
+    textareaReference.current.addEventListener('keydown', handleFormSubmitKey);
+
+    return () => textareaReference.current?.removeEventListener('keydown', handleFormSubmitKey);
+  }, []);
+
+  return (
+    <form onSubmit={handleFormSubmit} className="raf-comment-field">
+      <Avatar image={image} circle size={39} />
```

Comment:
rendering the default Avatar is a breaking change for customers css, make sure to update the PRs info with this and the dev branch once its merged

Reply:
I'll most likely return it to its previous state since this pattern appears in multiple other components and to stay consistent I'd have to announce breaking changes in all of those but I feel like that is not worth the hassle.